### ISSUE TITLE
'restart' don't work on Windows.

### ISSUE THREAD

**Issue by akaneko3** [deivid-rodriguez/byebug#319]
You say that Byebug support Windows, but 'restart' command isn't working on Windows.

```
(byebug) restart
Re exec'ing:
  C:/Ruby/lib/ruby/gems/2.3.0/gems/byebug-9.0.6/bin/byebug ...
*** No such file or directory - C:/Ruby/lib/ruby/gems/2.3.0/gems/byebug-9.0.6/bin/byebug
```

Actually, ```C:/Ruby/lib/ruby/gems/2.3.0/gems/byebug-9.0.6/bin/byebug``` is exist.

## Steps to reproduce the problem

```
>ruby -v
ruby 2.3.3p222 (2016-11-21 revision 56859) [x64-mingw32]

>gem install byebug

>byebug foo.rb
...
...
(byebug) restart
```

**Comment by deivid-rodriguez** [comment]
Thanks for the report @akaneko3!

Since I probably won't get to this in a while, feel free to investigate the problem yourself. I think this should not be a hard one to track, probably something with Windows path handling.

**Comment by deivid-rodriguez** [comment]
Closed by #321
